panic: Wrapper limit cannot be less than 1.

**Which version of kube-score are you using?**

> kube-score version: v1.2.0

**What did you do?**

> A complete description of the steps you took to trigger the incorrect behaviour.
> Include Kubernetes object YAML/JSON if you have it

**What did you expect to see?**

> A clear and concise description of what you expected to happen.

**What did you see instead?**

```
panic: Wrapper limit cannot be less than 1.

goroutine 1 [running]:
github.com/eidolon/wordwrap.Wrapper(0xfffffffffffffff3, 0xc000434600, 0xc0001f5608)
	/Users/gustav/go/pkg/mod/github.com/eidolon/wordwrap@v0.0.0-20161011182207-e0f54129b8bb/wordwrap.go:49 +0x83
main.outputHumanStep(0xeb7e22, 0xa, 0xc000403000, 0xa, 0xeb52b6, 0x3, 0xef08a1, 0x51, 0x0, 0x7, ...)
	/Users/gustav/src/kube-score/cmd/kube-score/main.go:292 +0x451
main.outputHuman(0xc000373728, 0x0, 0xffffffffffffffff, 0x101ba00, 0xc000403408)
	/Users/gustav/src/kube-score/cmd/kube-score/main.go:241 +0x6f6
main.scoreFiles(0xc00009a200, 0x0)
	/Users/gustav/src/kube-score/cmd/kube-score/main.go:173 +0x892
main.main()
	/Users/gustav/src/kube-score/cmd/kube-score/main.go:36 +0x2a0
```

--- a/cmd/kube-score/main.go
+++ b/cmd/kube-score/main.go
@@ -169,7 +169,11 @@ Use "-" as filename to read from STDIN.`)
 		w.WriteString(string(d))
 		r = w
 	} else if *outputFormat == "human" {
-		termWidth, _, _ := terminal.GetSize(int(os.Stdin.Fd()))
+		termWidth, _, err := terminal.GetSize(int(os.Stdin.Fd()))
+		// Assume a width of 80 if it can't be detected
+		if err != nil {
+			termWidth = 80
+		}
 		r = outputHuman(scoreCard, *verboseOutput, termWidth)
 	} else {
 		r = outputCi(scoreCard)
@@ -205,6 +209,13 @@ func listChecks() {
 	output.Flush()
 }
 
+func safeRepeat(s string, count int) string {
+	if count < 0 {
+		return ""
+	}
+	return strings.Repeat(s, count)
+}
+
 func outputHuman(scoreCard *scorecard.Scorecard, verboseOutput int, termWidth int) io.Reader {
 	// Print the items sorted by scorecard key
 	var keys []string
@@ -227,7 +238,7 @@ func outputHuman(scoreCard *scorecard.Scorecard, verboseOutput int, termWidth in
 		}
 
 		// Adjust to 80 columns wide
-		fmt.Fprintf(w, strings.Repeat(" ", 80-writtenHeaderChars-2))
+		fmt.Fprintf(w, safeRepeat(" ", 80-writtenHeaderChars-2))
 
 		if scoredObject.AnyBelowOrEqualToGrade(scorecard.GradeCritical) {
 			fmt.Fprintf(w, "💥\n")

--- a/cmd/kube-score/main.go
+++ b/cmd/kube-score/main.go
@@ -216,6 +216,13 @@ func safeRepeat(s string, count int) string {
 	return strings.Repeat(s, count)
 }
 
+func min(a, b int) int {
+	if a < b {
+		return a
+	}
+	return b
+}
+
 func outputHuman(scoreCard *scorecard.Scorecard, verboseOutput int, termWidth int) io.Reader {
 	// Print the items sorted by scorecard key
 	var keys []string
@@ -237,8 +244,8 @@ func outputHuman(scoreCard *scorecard.Scorecard, verboseOutput int, termWidth in
 			writtenHeaderChars += written2
 		}
 
-		// Adjust to 80 columns wide
-		fmt.Fprintf(w, safeRepeat(" ", 80-writtenHeaderChars-2))
+		// Adjust to termsize
+		fmt.Fprintf(w, safeRepeat(" ", min(80, termWidth)-writtenHeaderChars-2))
 
 		if scoredObject.AnyBelowOrEqualToGrade(scorecard.GradeCritical) {
 			fmt.Fprintf(w, "💥\n")
@@ -300,7 +307,11 @@ func outputHumanStep(card scorecard.TestScore, verboseOutput int, termWidth int)
 		fmt.Fprint(w, comment.Summary)
 
 		if len(comment.Description) > 0 {
-			wrapper := wordwrap.Wrapper(termWidth-12, false)
+			wrapWidth := termWidth - 12
+			if wrapWidth < 40 {
+				wrapWidth = 40
+			}
+			wrapper := wordwrap.Wrapper(wrapWidth, false)
 			wrapped := wrapper(comment.Description)
 			fmt.Fprintln(w)
 			fmt.Fprintf(w, wordwrap.Indent(wrapped, strings.Repeat(" ", 12), false))

--- a/cmd/kube-score/output_test.go
+++ b/cmd/kube-score/output_test.go
@@ -353,3 +353,51 @@ func TestHumanOutputLogDescription80Width(t *testing.T) {
             nisl venenatis, elementum augue a, porttitor libero.
 `, string(all))
 }
+
+func getTestCardLongTitle() *scorecard.Scorecard {
+	checks := []scorecard.TestScore{
+		{
+			Check: domain.Check{
+				Name: "test-warning-two-comments",
+			},
+			Grade: scorecard.GradeWarning,
+			Comments: []scorecard.TestScoreComment{
+				{
+					Path:        "a",
+					Summary:     "summary",
+					Description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras elementum sagittis lacus, a dictum tortor lobortis vel. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Nulla eu neque erat. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Maecenas et nisl venenatis, elementum augue a, porttitor libero.",
+				},
+			},
+		},
+	}
+
+	return &scorecard.Scorecard{
+		"a": &scorecard.ScoredObject{
+			TypeMeta: v1.TypeMeta{
+				Kind:       "Testing",
+				APIVersion: "v1",
+			},
+			ObjectMeta: v1.ObjectMeta{
+				Name:      "this-is-a-very-long-title-this-is-a-very-long-title-this-is-a-very-long-title-this-is-a-very-long-title-this-is-a-very-long-title",
+				Namespace: "foofoo",
+			},
+			Checks: checks,
+		},
+	}
+}
+
+func TestHumanOutputWithLongObjectNames(t *testing.T) {
+	r := outputHuman(getTestCardLongTitle(), 0, 80)
+	all, err := ioutil.ReadAll(r)
+	assert.Nil(t, err)
+	assert.Equal(t, `v1/Testing this-is-a-very-long-title-this-is-a-very-long-title-this-is-a-very-long-title-this-is-a-very-long-title-this-is-a-very-long-title in foofoo🤔
+    [WARNING] test-warning-two-comments
+        · a -> summary
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+            elementum sagittis lacus, a dictum tortor lobortis vel. Pellentesque
+            habitant morbi tristique senectus et netus et malesuada fames ac
+            turpis egestas. Nulla eu neque erat. Vestibulum ante ipsum primis in
+            faucibus orci luctus et ultrices posuere cubilia Curae; Maecenas et
+            nisl venenatis, elementum augue a, porttitor libero.
+`, string(all))
+}

--- a/cmd/kube-score/output_test.go
+++ b/cmd/kube-score/output_test.go
@@ -354,6 +354,26 @@ func TestHumanOutputLogDescription80Width(t *testing.T) {
 `, string(all))
 }
 
+func TestHumanOutputLogDescription0Width(t *testing.T) {
+	r := outputHuman(getTestCardLongDescription(), 0, 0)
+	all, err := ioutil.ReadAll(r)
+	assert.Nil(t, err)
+	assert.Equal(t, `v1/Testing foo in foofoo🤔
+    [WARNING] test-warning-two-comments
+        · a -> summary
+            Lorem ipsum dolor sit amet, consectetur
+            adipiscing elit. Cras elementum sagittis
+            lacus, a dictum tortor lobortis vel.
+            Pellentesque habitant morbi tristique
+            senectus et netus et malesuada fames ac
+            turpis egestas. Nulla eu neque erat.
+            Vestibulum ante ipsum primis in faucibus
+            orci luctus et ultrices posuere cubilia
+            Curae; Maecenas et nisl venenatis,
+            elementum augue a, porttitor libero.
+`, string(all))
+}
+
 func getTestCardLongTitle() *scorecard.Scorecard {
 	checks := []scorecard.TestScore{
 		{

